src/Entity/TagManager.php

```diff
@@ -36,12 +36,12 @@ public function getBySlug(string $slug, $context = null, ?bool $enabled = true):
         return $queryBuilder->getQuery()->getOneOrNullResult();
     }
 
-    public function getByContext($context, ?bool $enabled = true): array
+    public function getByContext(string $contextId, ?bool $enabled = true): array
     {
         $queryBuilder = $this->getRepository()
             ->createQueryBuilder('t')
             ->select('t')
-            ->andWhere('t.context = :context')->setParameter('context', $context);
+            ->andWhere('t.context = :context')->setParameter('context', $contextId);
```

Comment:
It is pedantic but will be awesome to change it too.
```suggestion
            ->andWhere('t.context = :contextId')->setParameter('contextId', $contextId);
```